corehq/apps/translations/app_translations/upload_module.py

```diff
@@ -100,7 +100,7 @@ def update(self, rows):
         if self.description:
             self._update_translation(self.description, self.module.search_config.description)
 
-        if self.no_items_text:
+        if self.no_items_text and hasattr(self.module, 'case_details'):
```

Comment:
This one isn't necessary because of the `if isinstance(self.module, ReportModule):` earlier in the function (demonstrating that we aren't consistent about how we do these checks...).

Reply:
Ok, I was thinking it would be a safeguard if there's other modules other than ReportModule that does't contain case_details

Reply:
removed - [7699629](https://github.com/dimagi/commcare-hq/pull/32602/commits/76996290cab5f9922946fbe261d6b44eeb88dbce)